physical/consul.go

```diff
@@ -548,22 +556,9 @@ func (c *ConsulBackend) reconcileConsul(registeredServiceID string, activeFunc a
 		}
 	}
 
-	tags := serviceTags(active)
-
-	var reregister bool
```

Comment:
It seems to me like we don't want to give up this logic, although the existing logic could be improved in several ways.